CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to USWDS web components
+
+Thank you for your interest in contributing to USWDS web components.
+
+> [!Important]
+>
+> **This repo is currently in beta and is not yet ready for community code contributions.**
+>
+> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
+>
+> Learn more about how you can provide feedback in the [submitting an issue](#submitting-an-issue) and [starting a discussion](#starting-a-discussion) sections below.
+
+## How can I contribute?
+
+### Submitting an issue
+
+If you have discovered a bug, let us know by [submitting an issue on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). We’ll address it as soon as we are able (but because we are in beta, please be aware that it might be awhile).
```

Comment:
**suggestion:** For convenience you could post the issue link here with a footnote on how to create issue.

Especially since the note at the top has similar language and links to our repo issue template.

```suggestion
If you have discovered a bug, let us know by [submitting an issue on GitHub](https://github.com/uswds/web-components/blob/al/add-contributing.md/CONTRIBUTING.md#submitting-an-issue) [^1]. 

_We’ll address it as soon as we are able (but because we are in beta, please be aware that it might be awhile)._

[^1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue
```

Reply:
Good notes!

I updated the link locations here to point to our repo rather than "how to" instructions from github.

Also, I ended up opting out of including a footnote because it felt unnecessary to me. The repo links feel pretty intuitive, and the footnotes also felt like they weren't quite right. If we do want to include links to github's "how to" guides, I lean towards making them plain text - something like "Learn more about submitting a GitHub issue..." 

Finally, I removed the on-page anchor links from the "important" note because they did have similar language, and ultimately felt unnecessary. Let me know if you have any objections or other thoughts here.